web/themes/custom/westy/package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "westy",
```

Comment:
Did you use your local npm installation instead of lando, by chance? I remember that causing this before. When I run `lando npm install` it reverts these changes.